### DISCUSSION TITLE
After updating to the latest, it keeps showing the following errors, any ideas? Thanks

### DISCUSSION THREAD

**Discussion by jzs11** [vueuse/vueuse#731] [Q&A]
"dependencies": {
    "@vuelidate/core": "^2.0.0-alpha.14",
    "@vuelidate/validators": "^2.0.0-alpha.13",
    "@vueuse/core": "^6.3.2",
    "@vueuse/integrations": "^6.3.2",
    "axios": "^0.21.1",
    "chart.js": "^3.5.0",
    "core-js": "^3.12.1",
    "devextreme": "^21.1.4",
    "devextreme-aspnet-data-nojquery": "^2.8.4",
    "devextreme-vue": "^21.1.4",
    "lodash": "^4.17.21",
    "primeflex": "^3.0.0",
    "primeicons": "^4.1.0",
    "primevue": "^3.5.1",
    "prismjs": "^1.15.0",
    "universal-cookie": "4.0.4",
    "vue": "^3.0.0",
    "vue-router": "^4.0.0-0",
    "vuedraggable": "^4.1.0"
  },

![image](https://user-images.githubusercontent.com/2770401/132655525-95a2506b-b00d-42c4-8f76-7cfb9355b7c2.png)

**Comment by antfu** [comment]
https://github.com/vueuse/vueuse/issues/718

**Reply by jzs11** [reply]
I tried the recommendation, working now. This is a regression right?

**Reply by antfu** [reply]
This is not a regression. We changed to `.mjs` and `.cjs` for future proof and Webpack 4 does not understand it out-of-box.

**Reply by jzs11** [reply]
Gotcha, thanks for the update.
